ticket.go

```diff
@@ -181,7 +191,7 @@ func (c *Conn) decryptTicket(encrypted []byte) (plaintext []byte, usedOldKey boo
 	macBytes := encrypted[len(encrypted)-sha256.Size:]
 	ciphertext := encrypted[ticketKeyNameLen+aes.BlockSize : len(encrypted)-sha256.Size]
 
-	keys := c.config.ticketKeys()
```

Comment:
Comment this line out instead of removing. Something like
// keys := c.config.ticketKeys() // [uTLS] keys are received as function argument

Reply:
[uTLS] is preferable